src/Workspaces/Core/Portable/Rename/RenameLocations.cs

```diff
@@ -115,8 +96,8 @@ private class SearchResult
         }
 
         public ISet<RenameLocation> Locations => _mergedResult.Locations;
-        public ISymbol Symbol => _symbol;
```

Comment:
are these set anywhere other than the constructors?

Reply:
they are not :)